addons/compat_spe/compat_spe_csw/CfgVehicles.hpp

```diff
@@ -9,53 +9,235 @@ class CfgVehicles {
         class ACE_Actions {
             class ACE_MainActions;
         };
-        class Turrets {
-            class MainTurret;
+    };
+
+    // --- Gun Turrets -------------------------------------------------------------
+    class StaticMGWeapon: StaticWeapon {};
+    class SPE_StaticMGWeapon_base: StaticMGWeapon {};
+    class SPE_M2_M3: SPE_StaticMGWeapon_base {
+        class ACE_Actions: ACE_Actions {
+            class ACE_MainActions: ACE_MainActions {
+                position = "";
+                selection = "zamerny";
+            };
+        };
+        class ace_csw {
+            enabled = 1;
+            proxyWeapon = "SPE_M2_M3_Tripod_proxy";
+            magazineLocation = "";
+            disassembleWeapon = "SPE_M2_50";
+            disassembleTurret = QGVAR(M3_Tripod);
+            ammoLoadTime = 3;
+            ammoUnloadTime = 3;
+        };
```

Comment:
most need a `desiredAmmo` or the gun will start empty
```suggestion
            desiredAmmo =100;
        };
```